### PR TITLE
Issue2578 simplyfication suggestion

### PR DESCRIPTION
**Pull request by uweschaefer**

### CODE REVIEW

File: factcast-client-grpc/src/test/java/org/factcast/client/grpc/GrpcFactStoreTest.java [hunk 1]
```diff
@@ -242,10 +243,40 @@ void testSerialOf() {
     assertNotSame(seven, response);
   }
 
+  //  @Test
+  //  @SneakyThrows
+  //  void testReInitializeWhenNotPreviouslyInitialized() {
```
Comment (bedaka) [inline factcast-client-grpc/src/test/java/org/factcast/client/grpc/GrpcFactStoreTest.java:248]: This can be removed

**Review by bedaka — APPROVED** [review]
Thanks for the improvements. I'll approve this PR & execute final QC on ENT

**Comment by uweschaefer** [comment]
it looks like, we turned one test flaky:

```
INFO  [      factcast/factcast:latest] Container factcast/factcast:latest started in PT5.940821141S
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.573 s -- in org.factcast.itests.factus.client.DatabaseUnhealthyITest
[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    GrpcStoreResilienceITest.testConcurrentRetryBehaviorWithoutResponse:131 
Expecting value to be true but was false
[INFO] 
Error:  Tests run: 101, Failures: 1, Errors: 0, Skipped: 0
```

File: factcast-itests/factcast-itests-factus/src/test/java/org/factcast/itests/factus/client/GrpcStoreResilienceITest.java [hunk 2]
```diff
@@ -207,7 +207,7 @@ public void run() {
     // Reconnect with 3 concurrent threads
     CountDownLatch latch = new CountDownLatch(3);
     ExecutorService threads = Executors.newFixedThreadPool(3);
-    for (int i = 0; i <= 3; ++i) {
+    for (int i = 0; i < 3; ++i) {
```
Comment (uweschaefer) [inline factcast-itests/factcast-itests-factus/src/test/java/org/factcast/itests/factus/client/GrpcStoreResilienceITest.java:210]: intentional? @bedaka

**Pull request merged**